great_ape_safe/ape_api/rari.py

```diff
@@ -17,13 +17,21 @@ def ftoken_is_paused(self, ftoken_addr):
 
 
     def ftoken_pause(self, ftoken_addr):
+        ftoken = interface.IFToken(ftoken_addr, owner=self.safe.account)
         self.unitroller._setBorrowPaused(ftoken_addr, True)
+        if ftoken.reserveFactorMantissa() != 0:
+            ftoken._setReserveFactor(0)
         assert self.ftoken_is_paused(ftoken_addr)
+        assert ftoken.reserveFactor() == 0
```

Comment:
what is the rationale behind resetting the rf to `0`? in my mind this just creates problems when unpausing, because you have to pass the right rf again for every asset.